### PR TITLE
Setup for rehype plugins

### PR DESCRIPTION
**Pull request by maxmarkus**
Excluded `unified` specific markdown transformer business logic to an extra file `website/docs/src/services/markdown.service.js` and added a plugin-development testrunner for quicker development.
Added also some nice rehype plugins which we can make use of.
Added boilerplate for custom plugin which will be followed up in ticket #919 for customization of a hrefs.

### CODE REVIEW

File: website/docs/INTERNAL-README.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ Production build:
 
 Open [http://localhost:5000](http://localhost:5000), it is exported to `public` folder and the micro frontend is stored in subfolder `docu-microfrontend`.
 
+## Developing markdown parser plugins
+
+Markdown are transformed into html using unified/remark2rehype, which has a already an ecosystem of plugins (npm packages are prefixed with rehype-). In order to quickly develop new plugins, a small script executes only the `src/services/markdown.service.js` file with a small markdown dataset defined in `scripts/markdown.unified-preview.js`. The referenced md files are located in `scripts/mocks`.
```
Comment (alexandra-simeonova) [inline website/docs/INTERNAL-README.md]:
```suggestion
Markdown files are transformed into html using unified/remark2rehype, which already has an ecosystem of plugins (npm packages are prefixed with rehype-). In order to quickly develop new plugins, a small script executes only the `src/services/markdown.service.js` file with a small markdown dataset defined in `scripts/markdown.unified-preview.js`. The referenced md files are located in `scripts/mocks`.
```

File: website/docs/scripts/mocks/plain.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Hello Luigi
+
+A standard file with some links, but not any special features
```
Comment (alexandra-simeonova) [inline website/docs/scripts/mocks/plain.md]:
```suggestion
`Hello Luigi` is a standard file with some links, but no special features.
```

File: website/docs/scripts/mocks/plain.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Choose the framework to build your application:
+
+[Application setup without a framework](#noframework) <br>
+[Angular 6](#angular6)<br>
+
+Follow these steps to build a web application based on Luigi:
+
+1. Add Luigi npm packages to your project dependencies.
```
Comment (alexandra-simeonova) [inline website/docs/scripts/mocks/plain.md]:
```suggestion
```

File: website/docs/scripts/mocks/plain.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Hello Luigi
+
+A standard file with some links, but not any special features
+
+Choose the framework to build your application:
+
+[Application setup without a framework](#noframework) <br>
+[Angular 6](#angular6)<br>
+
+Follow these steps to build a web application based on Luigi:
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply website/docs/scripts/mocks/plain.md]: Well, this file is anyway just a mock file, not any real type of documentation. Maybe I should just add some "lorem ipsum" there.

[the remark this answers is not in the document]
Comment (alexandra-simeonova) [reply website/docs/scripts/mocks/plain.md]: Oh, ok! No problem then 🙂

**Pull request merged**